sass/Pages/_photographer.scss

```
@@ -0,0 +1,326 @@
+    justify-content: space-between;
+    padding: 35px;
+    margin: auto;
+    margin-top: 3rem;
+
+    header {
+        justify-content: space-between;
+        width: 100%;
+        height: auto;
+
+        img {
+            cursor: pointer;
+            align-self: flex-start;
+        }
+        h2 {
+            font-size: 3.5rem;
+            font-weight: normal;
+        }
+    }
+
+    form {
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+        align-items: flex-start;
+    
+        label {
+            color: #312E2E;
+            font-size: 2rem;
+        }
+        div {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            align-items: self-start;
+            margin: 0.3rem 0;
+        }
+        input {
+            width: 100%;
+            height: 3rem;
+            border: none;
+            border-radius: 5px;
+            font-size: 1.5rem;
+        }
+        textarea{
+            width: 100%;
+            border: none;
+            border-radius: 5px;
+            font-size: 1.5rem;
+        }
+    }
+}
+
+
+.photo_section{
+    margin: 50px 100px;
+
+    .filter-form{
+
+        label{
+            font-size: 18px;
+            font-weight: 600;
+        
+        }
+        select {
+            font-size: 20px;
+            font-weight: 600;
+            color: white;
+            padding: 11px;
+            width: 170px;
+            /* height: 70px; */
+            border: none;
+            background-color: $primary;
+            border-radius: 5px;
+            cursor: pointer;
+            margin: 0 20px;
+        }
+
+        // .arrow{
+        //     transition: all 0.125s ease-in-out 0s;
+        //     transform: rotate(180deg);
+        // } 
+
+    }
+}
+
+.cards{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 70px;
+    margin-top: 50px;
+
+    article{
+        height: 40vh;
+        width: 20vw;
+        border-radius: 5px;
+    }
+
+    img {
+        height: 90%;
+        width: 100%;
+        object-fit: cover;
+        border-radius: 5px;
+    }
+    video{
+        height: 90%;
+        width: 100%;
+        object-fit: cover;
+        border-radius: 5px;
+    }
+
+    .card-info{
+        display: flex;
+        justify-content: space-between;
+
+        h2{
+            color: $primary;
+            font-weight: 500;
+            text-overflow: ellipsis;
+            overflow: hidden;
+            white-space: nowrap;
+        }
+
+        span{
+            align-self: center;
+            font-size: 18px;
+            font-weight: 600;
+            color: $primary;
+            margin: 3px;
+        }
+
+        .fas {
+            font-size: 18px;
+            align-self: center;
+            margin: 3px;
+            color:#DB8876;
+            cursor: pointer;
+            &:hover{
+                transform: scale(1.1);
+                color: $primary;
+            }
```

```suggestion
            &:hover, &.liked{
                  transform: scale(1.1);
                  color: $primary;
              }
```

ça marche ? (pour enlever les lignes du dessous)